ci/module-kmm-ci-build.template.yaml

```diff
@@ -23,9 +23,8 @@ spec:
               - name: build-secret
             dockerfileConfigMap:
               name: kmm-kmod-dockerfile
-            
             # Optional. If kanikoParams.tag is empty, the default value will be: 'latest'
             kanikoParams:
               tag: "debug"
   selector:
-    kubernetes.io/hostname: minikube
+    kubernetes.io/hostname: kmm-ci
```

Comment:
Do we have such a node on the test cluster?

Reply:
I pretended to label the node before applying:

https://github.com/rh-ecosystem-edge/kernel-module-management/pull/267/files#diff-d954caf0668f1d5539f11597234490aa6aadf4e53c4e175ba384b11244dbd21eR12

Reply:
I think we should use another label key then. `kubernetes.io/hostname` is managed by Kubernetes and might be overwritten.